examples/00_quick_start/fbt_movielens.ipynb

```diff
@@ -0,0 +1,770 @@
+{
```

Comment:
<p>it would be also good to test this notebook https://github.com/microsoft/recommenders/tree/main/tests#how-to-create-tests-on-notebooks-with-papermill</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>